src/_pytest/assertion/util.py

```diff
@@ -432,21 +444,34 @@ def _compare_eq_cls(
         else:
             diff.append(field)
 
-    explanation = []
+    explanation = [""]
```

Comment:
It will be slightly safer to change this to

```suggestion
    explanation = []
    if same or diff:
        explanation += [""]
```

Just in case sometime equal objects are passed in, we don't want the empty line.